src/Compilers/Core/CodeAnalysisTest/ObjectSerializationTests.cs

```diff
@@ -548,7 +548,7 @@ private static void TestWritingPrimitiveArrays(ObjectWriter writer)
             var inputString = new string[] { "h", "e", "l", "l", "o" };
 
             writer.WriteValue(inputBool);
-            writer.WriteValue(inputByte);
+            writer.WriteValue((object)inputByte);
```

Comment:
Consider adding a test for a large byte array on desktop framework

Reply:
Tests added for both net472 and netcoreapp3.1.